haste/windowed_conformal_model/conformal_interestingness_model.py

```diff
@@ -139,8 +141,8 @@ def interestingness(self,
 
             ts_features = numpy.append(correlation_ts_features, sum_of_intensities_ts_features, axis=0)
 
-
-            # p_values = conformal_interestingness(ALL_FEATURES, ts_features, ALL_Y)
+            ts_features_standardized = __standardize(ts_features, STANDARDIZING_VALUES[:, :, timestamp - 1])
```

Comment:
hmmm.. this will fail if the timestamp is too high (for another dataset). maybe we should check for that - and fallback to interestingness 1?

Reply:
wait a sec, that should be 
STANDARDIZING_VALUES[:, :, (timestamp / WINDOW_SIZE) - 1]
and similarly in the following part for the p_values change to
TRAIN_FEATURES_STANDARDIZED[:, :, (timestamp / WINDOW_SIZE) - 1]
sorry about that